src/common/constants.ts

```diff
@@ -43,21 +44,37 @@ export interface Network {
   chainId: ChainID;
 }
 
+export const MAINNET_MICROBLOCKS_SERVER = 'https://stacks-node-api.stacks.co';
+export const TESTNET_MICROBLOCKS_SERVER = 'https://stacks-node-api.testnet.stacks.co';
+export const REGTEST_MICROBLOCKS_SERVER = 'https://stacks-node-api.regtest.stacks.co';
+
+export const DEFAULT_TESTNET_SERVER = MICROBLOCKS_ENABLED
```

Comment:
I don't think you need to check this anymore, the url is the same?

Reply:
Yes indeed. I will remove it.